shader/antialiasing/cmaa2/compute_dispatch_args.comp

```diff
@@ -0,0 +1,50 @@
+#version 460
+
+#extension GL_ARB_separate_shader_objects       : enable
+#extension GL_GOOGLE_include_directive          : enable
+#extension GL_EXT_control_flow_attributes       : enable
+
+#include "cmaa2_common.glsl"
+
+layout(binding = 8) buffer UboWorkingExecuteIndirectBuffer {
+  uint g_workingExecuteIndirectBuffer[];
```

Comment:
Need to add explicit memory layout: `std430`.
Also why an array? Would it be nicer to have:
```
uvec3 shapeCandidateIndirectArg;
```
?